### DISCUSSION TITLE
create new replication using api not working

### DISCUSSION THREAD

**Discussion by ryzam** [apache/couchdb#3945] [Q&A] [answered]
Hi guys

I tried to create new continuous replication using HTTP API but unfortunately the replication execute only one time and it's not created in _replication database. 

`{
"user_ctx": {
    "name": "admin",
    "roles": [
      "_admin",
      "_reader",
      "_writer"
    ]
  },
  "source": {
    "url": "http://ubuntu-01.mshome.net:15984/master-todo",
    "headers": {
      "Authorization": "Basic xxxxxx"
    }
  },
  "target": {
    "url": "http://ubuntu-01.mshome.net:15984/branch1-todo",
    "headers": {
      "Authorization": "Basic xxxxxx"
    }
  },
  "selector": {
    "title": {
      "$regex": "^Fast"
    }
  },
  "create_target": false,
  "continuous": true,
  "owner": "admin"
}`

**Comment by nickva** [comment, marked as answer]
If a replication job is created via a  POST to `/_replicate` instead of via adding a document to a `.../_replicator` database, the first time the job crashes it will be evicted (removed) even if it is a continuous job. Try to use the a `_replicator` database instead, and create a replication document there.

Note: I edited the discussion question to turn the basic auth header details to into 'x'-s. In case those were not just toy / demo passwords and your actual credentials make sure to rotate them immediately, as they are now public.

**Comment by ryzam** [comment]
yes correct, I change the api endpoint from _replicate to _replicator then the document created in replicate database.

Tq.
